jekyll/_cci2/server-3-install-prerequisites.adoc

```diff
@@ -155,8 +155,6 @@ By default, CircleCI server will create self-signed certificates to get you star
 
 ==== Google Cloud DNS
 
-NOTE: In the following steps replace any items credentials displayed between `< >` with your details.
-
 . If you host your DNS on Google Cloud you will need the *certbot-dns-google* plugin installed. You can install the plugin with the following command: 
 +
 ```bash
```

Comment:
@circleci/server  Below where we have `<CIRCLECI_SERVER_DOMAIN>` is that the same as `<YOUR_CIRCLECI_NAMESPACE>` that we use in most places? I just want to keep this consistent if possible.

Reply:
No, server domain is the domain that is used to access the application

Reply:
I see, thanks!